core/store/migrate/migrations/0161_update_job_proposal_status.sql

```diff
@@ -0,0 +1,93 @@
+-- +goose Up
+-- +goose StatementBegin
+-- JobProposals Table
+-- We must remove the old contraint to add an enum value to support Postgres v11
+ALTER TABLE job_proposals DROP CONSTRAINT chk_job_proposals_status_fsm;
+
+ALTER TYPE job_proposal_status
+RENAME TO job_proposal_status_old;
+
+CREATE TYPE job_proposal_status AS ENUM(
+	'pending',
+	'approved',
+	'rejected',
+	'cancelled',
+	'deleted',
+	'revoked'
+);
+
+ALTER TABLE job_proposals
+ALTER COLUMN STATUS TYPE job_proposal_status USING STATUS::text::job_proposal_status;
```

Comment:
Better readability 

```suggestion
ALTER COLUMN status TYPE job_proposal_status USING STATUS::text::job_proposal_status;
```

Reply:
My sql formatter changed this on save, but I guess status is fine in postgres but a keyword in mysql